jenkins/Pipeline.gy

```diff
@@ -0,0 +1,11 @@
+pipeline {
+    agent { label 'pbench' }
+    stages {
+        stage('Linting & Unit Tests') {
+            steps {
+                echo 'Linting and legacy unit tests'
+                sh 'jenkins/run jenkins/tox -r --current-env -e jenkins-unittests'
```

Comment:
Instead of modifying `jenkins-unittests` to add explicit `black` and `flake8` invocations, why not just add the `lint` before `jenkins-unittests` tox environment target?

Reply:
Because, more or less, that's the way that `main` did it?  (I can do it whichever way we want...how do we want to do it?)

Reply:
I've changed `Pipeline.gy` to invoke the `lint` "environment" instead of running Black and Flake8 in the `jenkins-unittests` script.

Reply:
Ayaaaand, that didn't work.  (`tox` declines to install the linters before trying to run them, and after a couple of straightforward tweaks, I'm unable to convince it to change its mind.)  So, I've restored the flow to run the linters from the script, which executes after the installation.